src/Controller.cpp

```diff
@@ -12,4 +25,129 @@ Controller::~Controller() {
 
 void Controller::gameInput(Vehicle vehicle) {
 	//User input 
+}
+
+void Controller::processInput(float deltaSec)
+{
+	float rotationSpeed = glm::radians(135.0f) * deltaSec;
+	float scaleSpeed = 1.0f + 1.0f * deltaSec;
+
+	// Camera Movement
+	/*if (glfwGetKey(window, GLFW_KEY_W) == GLFW_PRESS)
+	{
+		camera->processKeyboard(Camera::Movement::FORWARD, deltaSec);
+	}
+
+	if (glfwGetKey(window, GLFW_KEY_S) == GLFW_PRESS)
+	{
+		camera->processKeyboard(Camera::Movement::BACKWARD, deltaSec);
+	}*/
```

Comment:
Can you delete the code (here and in some other files) rather than just commenting it out? If we start commenting out code it is gonna get messy really quick. 

And since we are using git we can always check the history if we want to see how it was done before.

Don't worry about the commented code that you didn't comment out, like the one Tylar added to Engine.cpp.